packages/table/src/TableCheckboxCell.ts

```diff
@@ -71,6 +72,7 @@ export class TableCheckboxCell extends SpectrumElement {
                 ?disabled=${this.disabled}
                 ?emphasized=${this.emphasized}
                 aria-hidden=${ifDefined(this.selectsSingle)}
+                aria-label="${this.ariaLabel || nothing}"
```

Comment:
Maybe we can just align on using
`ifDefined(this.ariaLabel || undefined)` OR `this.ariaLabel || nothing` 

I guess it feels inconsistent to me that here we are using `nothing` and above we are using the other variation

Reply:
I agree with you. The linter was complaining about the `IfDefined(this.ariaLabel)` and I found something in the lit document that was mentioning `this.ariaLabel || nothing` instead. I would rather keep the other variant, but I was unable to silence the linter

Reply:
https://lit.dev/docs/templates/conditionals/#conditionally-rendering-nothing